### PR TITLE
Controllers

### PR DESCRIPTION
**Pull request by MICHAELMUNAVU83**
- Use controllers to handle requests and render empty views.
- Understand Rails naming conventions.
- Use params from browser requests in a safe way.
- Use a user id of 2 AND a post id of 2 to test .

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :users do
+    resources :posts
+
+  end
+ 
+  root "users#index"
```
Comment (descholar-ceo) [inline config/routes.rb]: - [ ] You were supposed to do not expose unnecessary routes, kindly fix your routes as follow:
```suggestion
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @MICHAELMUNAVU83 🤝 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
👍🏽 PR doesn't have any linter's error ✔️ 
👍🏽 PR follows the correct git/github flows ✔️ 
👍🏽 The PR implements more than 90% of the requirements of this project ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (MICHAELMUNAVU83) [reply config/routes.rb]: Thank you very much

**Review by descholar-ceo — APPROVED** [review]
Hi @MICHAELMUNAVU83 🤝 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
👍🏽 Implemented all of the requested changes ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**